### ISSUE TITLE
A few issues/enhancements for map plots

### ISSUE THREAD

**Issue by mharty3** [PatrikHlobil/Pandas-Bokeh#61]
Hi Patrik,
Great library. I am a fan and have enjoyed using it so far. I have a few issues that might be able to help improve the geomap plots. I'd like to get your thoughts. I can split them up into different issues and submit PR's as well.

1. Add support for `scale_height` argument for the map plots to allow for responsive sizing of the map

2. The Reset Tool has unexpected behavior. It resets the aspect ratio of the maps, I would expect that the aspect ratio would be locked for maps. This can be solved by including `match_aspect=True` in the figure options.

https://github.com/PatrikHlobil/Pandas-Bokeh/blob/5e100396e857e91954970105bf9b43e4ccb11f50/pandas_bokeh/geoplot.py#L253-L262

3. Similar to the one above, I would expect the box zoom to lock the aspect ratio. The box zoom tool has a similar argument to accomplish this.`BoxZoomTool(match_aspect=True)`

**Comment by PatrikHlobil** [comment]
Hi @mharty3 ,

thanks a lot for your feedback. I already implemented your issue 2) and plan to do 3) now. 

Can you explain what you mean with your first issue?

Best Patrik

**Comment by mharty3** [comment]
Yes Patrik, sorry I typed the wrong thing for number 1. It should have said: 

1. Add support for `sizing_mode` argument for the map plots to allow for responsive sizing of the map

It looks like it was implemented for general plots, but not for maps.
https://github.com/PatrikHlobil/Pandas-Bokeh#auto-scaling-plots
